docs/minio-admin-complete-guide.md

```diff
@@ -0,0 +1,311 @@
+heal         Start an object heal operation
+```
+
+## 1.  Download Minio Client
+### Docker Stable
+```
+docker pull minio/mc
+docker run minio/mc admin info play
+```
+
+### Docker Edge
+```
+docker pull minio/mc:edge
+docker run minio/mc:edge admin info play
+```
+
+**Note:** Above examples run `mc` against Minio [_play_ environment](#test-your-setup) by default. To run `mc` against other S3 compatible servers, start the container this way:
+
+```sh
+docker run -it --entrypoint=/bin/sh minio/mc
+```
+
+then use the [`mc config` command](#add-a-cloud-storage-service).
+
+### Homebrew (macOS)
+Install mc packages using [Homebrew](http://brew.sh/)
+
+```sh
+brew install minio/stable/mc
+mc --help
+```
+
+### Binary Download (GNU/Linux)
+| Platform | Architecture | URL |
+| ---------- | -------- |------|
+|GNU/Linux|64-bit Intel|https://dl.minio.io/client/mc/release/linux-amd64/mc |
+
+```sh
+chmod +x mc
+./mc --help
+```
+
+### Binary Download (Microsoft Windows)
+| Platform | Architecture | URL |
+| ---------- | -------- |------|
+|Microsoft Windows|64-bit Intel|https://dl.minio.io/client/mc/release/windows-amd64/mc.exe |
+
+```sh
+mc.exe --help
+```
+
+### Install from Source
+Source installation is intended only for developers and advanced users. `mc update` command does not support update notifications for source based installations. Please download official releases from https://minio.io/downloads/#minio-client.
+
+If you do not have a working Golang environment, please follow [How to install Golang](https://docs.minio.io/docs/how-to-install-golang).
+
+```sh
+go get -d github.com/minio/mc
+cd ${GOPATH}/src/github.com/minio/mc
+make
+```
+
+## 2. Run Minio Client
+
+### GNU/Linux
+
+```sh
+chmod +x mc
+./mc --help
+```
+
+### macOS
+
+```sh
+chmod 755 mc
+./mc --help
+```
+
+### Microsoft Windows
+
+```sh
+mc.exe --help
+```
+
+## 3. Add a Minio Storage Service
+Minio server displays URL, access and secret keys.
+
+#### Usage
+
+```sh
+mc config host add <ALIAS> <YOUR-S3-ENDPOINT> <YOUR-ACCESS-KEY> <YOUR-SECRET-KEY>
+```
+
+Alias is simply a short name to you cloud storage service. S3 end-point, access and secret keys are supplied by your Minio service. Admin API uses "S3v4" signature and cannot be changed.
+
+```sh
+mc config host add minio http://192.168.1.51 BKIKJAA5BMMU2RHO6IBB V7f1CwQqAcwo80UEIJEjc5gVQUSSx5ohQ9GSrr12
+```
+
+## 4. Test Your Setup
+`mc` is pre-configured with https://play.minio.io:9000, aliased as "play". It is a hosted Minio server for testing and development purpose.
+
+*Example:*
+
+Get Minio server information from https://play.minio.io:9000
+
+```sh
+●  play.minio.io:9000
+   Uptime : online since 1 day ago
+  Version : 2018-05-28T04:31:38Z
+   Region :
+ SQS ARNs : <none>
+    Stats : Incoming 82GiB, Outgoing 28GiB
+  Storage : Used 7.4GiB
+```
+
+## 5. Everyday Use
+You may add shell aliases for info, healing.
+
+```sh
+alias minfo='mc admin info'
+alias mheal='mc admin heal'
+```
+
+## 6. Global Options
```

Comment:
IMHO steps 1 through 5 are repetitive from https://github.com/minio/mc/blob/master/docs/minio-client-complete-guide.md . We can probably refer the `mc complete guide` doc in the beginning here and start from step 6 in this doc.

Reply:
It is repeated on purpose since minio client complete guide repeats it from README.md

Reply:
okay, in that case we can probably take this and do a cleanup to avoid repetition in the `complete guides`.